synapse/config/api.py

```diff
@@ -59,6 +60,11 @@ def generate_config_section(cls, **kwargs) -> str:
            #
            #additional_event_types:
            #  - org.example.custom.event.type
+
+        # If enabled, puppeted user IP's can also be tracked. By default when
+        # puppeting another user, the user who has created the access token
+        # for puppeting is tracked. If this is enabled, both requests are tracked.
```

Comment:
`track_appservice_user_ips` has a note about mau tracking ("Implicitly enables MAU tracking for application service users."). Shall we add a similar note here?

Reply:
I thought I added that /o\ I was going to at least, I swear :sweat_smile: 

Added now!